src/libsystemd-network/ndisc-internal.h

```diff
@@ -47,6 +47,15 @@ struct sd_ndisc {
         void *userdata;
 };
 
+#define SD_RADV_DEFAULT_MIN_TIMEOUT             200 * USEC_PER_SEC
+#define SD_RADV_DEFAULT_MAX_TIMEOUT             600 * USEC_PER_SEC
```

Comment:
please enclose these values in `( )`, so that we don't run into operator precedence issues when they are used